packages/table/src/Cell/HeaderCell/HeaderCell.tsx

```diff
@@ -67,7 +67,7 @@ const HeaderCell = <T extends LGRowData>({
         className={cx(
           cellContentContainerStyles,
           headerCellContentStyles,
-          // @ts-expect-error Header types need to be extended or declared in the react-table namespace
+          // @ts-ignore Header types need to be extended or declared in the react-table namespace
```

Comment:
This should still be expect-error, so we can remove it if/when it's fixed

Reply:
This is one of those errors that doesn't show up locally, but shows up in build. So if we were to keep @ts-expect-error, we'd get an error for an unused directive.

Reply:
Updated comment accordingly!